rpcd/playbooks/roles/kibana/tasks/kibana_pre_install.yml

```diff
@@ -13,50 +13,40 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-- name: Install Kibana apt packages
-  apt:
-    pkg: "{{ item }}"
-    state: latest
-    update_cache: yes
-    cache_valid_time: 600
-  register: install_apt_packages
-  until: install_apt_packages|success
+- name: Add kibana apt-keys
+  apt_key:
+    url: "{{ item.url }}"
+    state: "{{ item.state }}"
+  with_items: kibana_apt_keys
```

Comment:
[So](https://github.com/rcbops/rpc-openstack/blob/c68e257057371e2ac68d0f78d74422af9f5e1944/rpcd/playbooks/roles/horizon_extensions/tasks/main.yml#L27), we used to write it the way I described. I'm glad this syntax works though and is now the de facto standard here. (The last time I tried to convert us to it, it was shot down.) I suspect Ansible 2.x has fixed the problems we had with it.

👍